ui/src/app/pages/workspace/featured-workspaces.tsx

```diff
@@ -0,0 +1,314 @@
+import * as React from 'react';
+import { ReactNode, useEffect, useState } from 'react';
+
+import { FeaturedWorkspaceCategory, WorkspaceResponse } from 'generated/fetch';
+
+import { AlertDanger } from 'app/components/alert';
+import { Clickable } from 'app/components/buttons';
+import { FlexColumn, FlexRow } from 'app/components/flex';
+import { Header } from 'app/components/headers';
+import { Spinner } from 'app/components/spinners';
+import { AoU } from 'app/components/text-wrappers';
+import { WorkspaceCard } from 'app/pages/workspace/workspace-card';
+import { featuredWorkspaceApi } from 'app/services/swagger-fetch-clients';
+import colors, { colorWithWhiteness } from 'app/styles/colors';
+import { reactStyles } from 'app/utils';
+import { hasTierAccess } from 'app/utils/access-tiers';
+import { convertAPIError } from 'app/utils/errors';
+import { profileStore } from 'app/utils/stores';
+import { WorkspacePermissions } from 'app/utils/workspace-permissions';
+import community from 'assets/icons/community.svg';
+import demonstration from 'assets/icons/demonstration.svg';
+import phenotypeLibrary from 'assets/icons/phenotype-library.svg';
+import tutorialWorkspaces from 'assets/icons/tutorial-workspaces.svg';
+
+const styles = reactStyles({
+  navPanel: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: '1.5rem',
+    backgroundColor: colorWithWhiteness(colors.primary, 0.85),
+    minWidth: '300px',
+    maxWidth: '300px',
+    marginLeft: '-0.9rem',
+  },
+  searchBar: {
+    height: '3rem',
+    fontSize: '16px',
+    lineHeight: '19px',
+    paddingLeft: '3rem',
+    marginTop: '0.75rem',
+    borderStyle: 'none',
+  },
+  iconStyling: {
+    color: colors.primary,
+    marginRight: '0.75rem',
+    marginLeft: '0.75rem',
+    height: 24,
+    width: 24,
+  },
+  menuLink: {
+    color: colors.accent,
+    fontSize: 14,
+    fontWeight: 600,
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    padding: '0.525rem 0',
+    borderRadius: '3px',
+  },
+  divider: {
+    width: '100%',
+    backgroundColor: colors.primary,
+    borderWidth: '0px',
+    height: '1px',
+    marginBottom: '0.75rem',
+  },
+  menuLinkSelected: {
+    backgroundColor: colorWithWhiteness(colors.primary, 0.95),
+  },
+  libraryTabDivider: {
+    width: '100%',
+    margin: '0.75rem 0',
+    backgroundColor: colorWithWhiteness(colors.dark, 0.5),
+    border: '0 none',
+    height: 1,
+  },
+});
+
+interface LibraryTabProps {
+  title: string;
+  description: ReactNode;
+  icon: string;
+  category: FeaturedWorkspaceCategory;
+}
+
+const createTab = (title, description, icon, category): LibraryTabProps => ({
```

Comment:
I'm not sure this indirection serves a purpose any more.  What do you think about defining the tabs inline instead? That seems clearer to me.

Like:
```
PHENOTYPE_LIBRARY: {
    title: 'Phenotype Library',
    description: <div>
      These workspaces demonstrate how computable electronic phenotypes can be
      implemented within the <AoU /> dataset using examples of previously
      published phenotype algorithms. You can open the workspaces to view them
      or “duplicate” the workspaces to edit and execute the algorithms.
    </div>,
    icon: phenotypeLibrary,
    category: FeaturedWorkspaceCategory.PHENOTYPE_LIBRARY
  },
```

Reply:
Yeah that makes sense